apps/web-giddh/src/app/vat-report/vat-report.component.ts

```diff
@@ -516,7 +534,7 @@ export class VatReportComponent implements OnInit, OnDestroy {
      */
     public onCurrencyChange(event: any): void {
         if (event) {
-            this.vatReportCurrency = event?.value;
+            this.vatReportCurrencyCode = event?.value;
```

Comment:
remove nulity check for event.value already check in if condition

Reply:
I checked only event, but as per your suggestion I update my code as 

 if (event?.value) {
            this.vatReportCurrencyCode = event.value;
            this.getVatReport();
        }

Reply:
I said remove only nulity check in event.value already you are check event in if condition